src/main/java/core/basesyntax/impl/CsvReportServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class CsvReportServiceImpl implements ReportService {
+    private static final String REPORT_TITLE = "fruit,quantity";
+    private static final String DELIMITER = ",";
+
+    @Override
+    public String createReport(Map<String, Integer> fruitStorage) {
```

Comment:
You didn't correct my comment. Don't wait for the map to be taken from the database (Storage)

Reply:
Hint - add  FruitDao as class-level variable.
```suggestion
    public String createReport() {
```